.github/workflows/build.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
-    - run: npm install
+    - run: npm ci
```

Comment:
Was not aware that this was a thing, but from a quick glance at the documentation, good call adding it!